aea/helpers/async_utils.py

```diff
@@ -477,3 +458,206 @@ def __init__(self, getters: List[Tuple[Callable[[Any], None], Callable]]):
         super().__init__(
             [self._make_getter(handler, getter) for handler, getter in getters]
         )
+
+
+ready_future: Future = Future()
+ready_future.set_result(None)
+
+
+class Runnable(ABC):
+    """Abstract Runnable class."""
+
+    def __init__(self, loop=None, threaded=False) -> None:
+        """
+        Init runnable.
```

Comment:
Explain a bit more in the docstring what the purpose of the class is

Reply:
done